src/views/product-root-docs-path-landing/server.ts

```diff
@@ -81,10 +83,19 @@ const generateHeadingLevelsAndSidecarHeadings = ({
 
 const generateGetStaticProps = ({
 	includeMDXSource = false,
-	pageContent,
 	product,
 	productSlugForLoader,
 }: GenerateGetStaticPropsArguments) => {
+	/**
+	 * Note: could consider other content sources. For now, JSON.
+	 * Asana task: https://app.asana.com/0/1100423001970639/1201631159784193/f
+	 */
+	const jsonFilePath = path.join(
+		process.cwd(),
+		`src/content/${product.slug}/docs-landing.json`
+	)
+	const pageContent = JSON.parse(fs.readFileSync(jsonFilePath, 'utf8'))
```

Comment:
🗒️ Same as previous comment, this updates the content to be loaded from a file path. And the file path is generated from the given product's slug.